Mongo starting, done?

mavridaki opened the pull request.

mavridaki committed:
Response RAPPER

Implemented all methods with ResponseWrapper, everythings works fine!!!
https://www.youtube.com/watch?v=Hw6RFGzbc5Y

mavridaki committed:
Some changes in connectMongo and MongoTesting

mavridaki committed:
MongoTesting

A small step for MongoTesting,
a giant leap for EESTEC LC Thessaloniki

mavroudo commented:
Heyyy!!  Really  nice documented code! I want you to do the following:
1) In file connectMongo.py :
 a) Remove function getAgendaJsonById that was never used
 b) Replace every code that does search in mongo by id, with the function getAgendaById (makes code easier to read and easier to maintain if for example in a next patch we want to search with more parameters )
2) Tests are wrong. You have to implement what you want to be checked inside every method, like in AgendaTesting. If you have problem tell me

mavridaki committed:
Pull request changes

Replaced every code that does search in mongo by id, with the function getAgendaById

mavridaki committed:
UpdateSection, UpdateTopic

What you asked for, Senpai
